src/shared/components/event-management/event-start/event-start.component.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { CUSTOM_ELEMENTS_SCHEMA, DebugElement } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute } from '@angular/router';
+import { EventStartComponent } from './event-start.component';
+import { EventStateMachineService } from '../services/event-state-machine.service';
+import createSpyObj = jasmine.createSpyObj;
+import { EventStates } from '../models';
```

Comment:
Minor comment - Ordering issues again here which will cause lint warnings

Reply:
Thank you @connorpgpmcelroy, I used the command Shift + Alt + O to organise the imports.